template/pages/@/views/products.ejs

```diff
@@ -97,7 +101,7 @@ breadcrumbs.push({
             items = search.getItems()
           }
           %>
-          <%- await include('components/products-carousel', { _, opt: { items, title } }) %>
+          <%- await include('components/products-carousel', { _, opt: { items, title, showcaseType } }) %>
```

Comment:
Vai ter o mesmo problema que você teve na pitbar antes, se o `products.reated.showcase` for falso, não vai incluir o componente e portanto ele não vai estar no output HTML e não será reativo no preview...

Reply:
Mas neste caso o buraco é um pouco mais embaixo porque renderizar ou não o carousel nas páginas de produto faz uma diferença bem considerável no tempo de build.
Talvez se você mockar (com produtos mockados mesmo) uma div dentro de `<script data-cms-html>` que lembre mais ou menos o carousel e utilize isso no preview?